docs/architecture/seo.md

```diff
@@ -1,46 +1,47 @@
 # SEO Capabilities
-As a storefront framework, Spartacus cares about Search Engine Optimisation. This is a large topic but main focus for Spartacus has been on the architectural side so that we provide a solid foundation that allows for SEO capabilities rather than implement all kinds of features. 
 
-We're covering the following topics: 
+Search engine optimization (SEO) is an important element of the Spartacus storefront framework. The implementation of SEO in Spartacus is focused on the architecture, rather than on features, so that we can provide a solid foundation that allows for a wide range of SEO capabilities.
```

Comment:
I reworked it a little more to say:

"The implementation of SEO in Spartacus is focused on the architecture, rather than on features, so that a solid foundation exists for adding a wide range of SEO capabilities."

Does this work?